torchmd/npzMol.py

```diff
@@ -1,33 +1,28 @@
 import numpy as np
 import torch
+from moleculekit.periodictable import periodictable_by_number
 
 
 class npzMolecule:
-    """Class to load npz files in torchmd, essentially a wrapper around the npz file. Necessary information is extracted from the npz file
-    and stored in the class, such as the coordinates and the atomic numbers. The class also contains a dictionary to convert atomic numbers to
-    atomic symbols. Additional properties such as the box, the charges and the bonds are also stored in the class but are not necessary for the
-    simulation."""
-
     def __init__(self, file: str):
-        self.converter = {
-            "1": ["H", 1.0080],
-            "6": ["C", 12.011],
-            "7": ["N", 14.007],
-            "8": ["O", 15.999],
-            "9": ["F", 18.99840316],
-            "15": ["P", 30.97376200],
-            "16": ["S", 32.07],
-            "17": ["Cl", 35.45],
-            "35": ["Br", 79.90],
-            "53": ["I", 126.9045],
-        }
+        """Class to load npz files in torchmd, essentially a wrapper around the npz file. Necessary information is extracted from the npz file
+        and stored in the class, such as the coordinates and the atomic numbers. Additional properties such as the box, the charges and the bonds
+        are also stored in the class but are not necessary for the simulation."""
+
         self.data = np.load(file)
-        self.z = torch.from_numpy(self.data["z"])
-        self.coords = torch.from_numpy(self.data["coord"])[:, :, None]
+        self.z = np.array(self.data["z"])
+        self.coords = np.array(self.data["coord"])[:, :, None]
```

Comment:
I don't understand why here you stopped converting to torch tensor and just make numpy array and then `masses` two lines down is converted to torch tensor. Shouldn't they all be either numpy arrays or torch tensors?

Reply:
The npz mol class is building a mol object similar to the one from moleculekit.molecule so in this case we don't need the tensor but just numpy array that will transformed later to tensor. 

About the masses: the original idea from which this class came out was to run torchmd using only `pos` and `z`, no ff object and so this class returns directly a tensor for the masses because then there is this  [if](https://github.com/torchmd/torchmd/blob/1bad86ec08cec101320d66b24529a032ace4d016/torchmd/parameters.py#L116) that let you to skip make_masses() since it needs ff. Maybe we can remove it or put it as np.array, since now we are considering to use the  mol.element #61

Reply:
Also using elements instead of FF masses is not good because they can be different and will affect the results of the simulations.

Reply:
I wrote that `if` probably when I wrote the npzMol class.